src/studio/src/designer/frontend/ux-editor/components/config/HeaderSizeSelect.tsx

```diff
@@ -12,15 +12,22 @@ export interface HeaderSizeSelectProps {
   handleUpdateHeaderSize: (e: any) => void
 }
 
+const sizeCompat:any = {
```

Comment:
Use enum instead?

```typescript
enum sizeCompat {
  S = 'h4',
  M = 'h3',
  L = 'h2',
};
```

Reply:
It needs to be an object runtime, and I had to do `:any` in order to be able to index with the unknown string. Does an enum work with indexing?

Reply:
Why does it need to be an object runtime? 

Using an enum gives a bit nicer TS intellisense, and its clear that its a enum type. When compiling, it will create a function which returns the correct value.